packages/core/core/src/plugins.ts

```diff
@@ -149,7 +160,9 @@ export class Plugins {
       fs: this.fs,
       isInternal: false,
       context: this.context,
-      version: this.version
+      version: this.version,
+      client: this.client,
+      locale: this.locale
```

Comment:
Again, just reiterating that we don't need the changes here, since `locale` and `client` are both accessible in `config`